src/backend/aspen/cli/db.py

```diff
@@ -198,7 +198,7 @@ def add_can_see(
     "--builds-template-file",
     type=str,
     required=False,
-    default="src/backend/aspen/workflows/nextstrain_run/builds_templates/group.yaml",
+    default="src/backend/aspen/workflows/nextstrain_run/builds_templates/group_plus_context.yaml",
```

Comment:
what is this used for? (purely out of curiousity)

Reply:
it technically doesn't get used at all - it's the default value to use if a --builds-template-file argument isn't specified (and we always specify one.) But referring to a missing file just seemed like a bad idea 🤷‍♀️